tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -863,7 +863,7 @@ def first_location_of_minimum(x):
 
 
 @set_property("fctype", "simple")
-def percentage_of_reoccurring_datapoints_to_all_datapoints(x):
+def percentage_of_reoccurring_values_to_all_values(x):
     """
     Returns the percentage of unique values, that are present in the time series
```

Comment:
I think the function descriptions are misleading. I would change it to

```suggestion
    Returns the percentage of values in the time series, that are present 
```